src/ir_Sony.cpp

```diff
@@ -46,10 +51,50 @@ const uint16_t kSonyMinGap = kSonyMinGapTicks * kSonyTick;
 // Ref:
 //   http://www.sbprojects.com/knowledge/ir/sirc.php
 void IRsend::sendSony(uint64_t data, uint16_t nbits, uint16_t repeat) {
```

Comment:
Should the default for repeat be explicitly set? ie: `uint16_t repeat = 3`

Reply:
No; & Yes, it is!
See:
https://github.com/crankyoldgit/IRremoteESP8266/blob/Sony38kHz/src/IRsend.h#L205 & https://github.com/crankyoldgit/IRremoteESP8266/blob/Sony38kHz/src/IRremoteESP8266.h#L833

For `sendSony()` the default `repeat` is `2`. For `sendSony38()` the default `repeat` is `3` (based on user observation).

Reply:
Agg! Well done :) 
(Not the first time I've done that either) Cheers.